test/llvm_autogenerated/cfg-stackify.wast.orig

```diff
@@ -0,0 +1,1128 @@
+(module
+  (memory $0 1)
+  (data (i32.const 4) "\10\04\00\00")
+  (type $FUNCSIG$v (func))
+  (type $FUNCSIG$i (func (result i32)))
+  (import "env" "a" (func $a (result i32)))
+  (import "env" "bar" (func $bar))
+  (import "env" "something" (func $something))
+  (import "env" "test15_callee0" (func $test15_callee0))
+  (import "env" "test15_callee1" (func $test15_callee1))
+  (export "memory" (memory $0))
+  (export "test0" (func $test0))
+  (export "test1" (func $test1))
+  (export "test2" (func $test2))
+  (export "doublediamond" (func $doublediamond))
+  (export "triangle" (func $triangle))
+  (export "diamond" (func $diamond))
+  (export "single_block" (func $single_block))
+  (export "minimal_loop" (func $minimal_loop))
+  (export "simple_loop" (func $simple_loop))
+  (export "doubletriangle" (func $doubletriangle))
+  (export "ifelse_earlyexits" (func $ifelse_earlyexits))
+  (export "doublediamond_in_a_loop" (func $doublediamond_in_a_loop))
+  (export "test3" (func $test3))
+  (export "test4" (func $test4))
+  (export "test5" (func $test5))
+  (export "test6" (func $test6))
+  (export "test7" (func $test7))
+  (export "test8" (func $test8))
+  (export "test9" (func $test9))
+  (export "test10" (func $test10))
+  (export "test11" (func $test11))
+  (export "test12" (func $test12))
+  (export "test13" (func $test13))
+  (export "test14" (func $test14))
+  (export "test15" (func $test15))
+  (table 0 anyfunc)
+  
+  (func $test0 (param $0 i32)
+    (local $1 i32)
+    (set_local $1
+      (i32.const 1)
+    )
+<<<<<<< HEAD
+    (loop $label$0
+      (block $label$1
+        (br_if $label$1
+          (i32.lt_s
+            (get_local $1)
+            (get_local $0)
+=======
+    (block $label$1
+      (loop $label$0
+        (block $label$2
+          (br_if $label$2
+            (i32.lt_s
+              (get_local $1)
+              (get_local $0)
+            )
+>>>>>>> origin/master
+          )
+          (return)
+        )
+        (return)
+      )
+      (set_local $1
+        (i32.add
+          (get_local $1)
+          (i32.const 1)
+        )
+      )
+      (call $something)
+      (br $label$0)
+    )
+  )
+  (func $test1 (param $0 i32)
+    (local $1 i32)
+    (set_local $1
+      (i32.const 1)
+    )
+<<<<<<< HEAD
```

Comment:
Merge conflicted here, and in 15 more places

Looks to be only in the autogenerated tests though, so probably worth just regenerating them now that this is rebased